Instrumentation fails on `pdfbox` classes, even though jar with the missed class is on classpath

Consider the following tests generation scenario:

```java
timeLimit = 120
methodFilter = "org.apache.pdfbox.util.XMLUtil.*"
projectFilter = listOf("pdfbox")
```

The following exception occured:

```
java.lang.Exception: java.lang.Exception: Unexpected exception in endpoint `usvm-executor-worker.InstrumentedProcessModel.callUTest`::(9997603787970205307) taskId=(1000378)
	at com.jetbrains.rd.framework.impl.RdCall.onWireReceived(RdTask.kt:371)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:57)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:56)
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:148)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:56)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:54)
	at com.jetbrains.rd.util.threading.SingleThreadSchedulerBase.queue$lambda-3(SingleThreadScheduler.kt:41)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.jacodb.api.NoClassInClasspathException: Class com.github.jaiimageio.impl.plugins.tiff.TIFFFieldNode not found in classpath
	at org.jacodb.api.ExceptionsKt.throwClassNotFound(Exceptions.kt:26)
	at org.jacodb.api.ext.JcClasspaths.findClass(JcClasspaths.kt:51)
	at org.usvm.instrumentation.serializer.SerializationUtilsKt.readJcClass(SerializationUtils.kt:39)
	at org.usvm.instrumentation.serializer.UTestInstSerializer.deserializeUTestAllocateMemoryCall(UTestInstSerializer.kt:256)
	at org.usvm.instrumentation.serializer.UTestInstSerializer.deserializeUTestInstFromBuffer(UTestInstSerializer.kt:82)
	at org.usvm.instrumentation.serializer.UTestInstSerializer.deserializeUTestInst(UTestInstSerializer.kt:102)
	at org.usvm.instrumentation.serializer.UTestInstSerializer$Companion$marshaller$$inlined$create$1.invoke(FrameworkMarshallers.kt:152)
	at org.usvm.instrumentation.serializer.UTestInstSerializer$Companion$marshaller$$inlined$create$1.invoke(FrameworkMarshallers.kt:31)
	at com.jetbrains.rd.framework.UniversalMarshaller.read(FrameworkMarshallers.kt:14)
	at org.usvm.instrumentation.generated.models.SerializedUTest$Companion.read(InstrumentedProcessModel.Generated.kt:490)
	at org.usvm.instrumentation.generated.models.SerializedUTest$Companion.read(InstrumentedProcessModel.Generated.kt:485)
	at com.jetbrains.rd.framework.impl.RdCall.onWireReceived(RdTask.kt:355)
	... 9 more
```

However, the class `com.github.jaiimageio.impl.plugins.tiff.TIFFFieldNode` is present on classpath, it is shown on the screen.
This problem happens for some other classes too.


![image](https://github.com/UnitTestBot/usvm/assets/29856383/0bf04620-7b07-4a69-88b0-da1c82773dd1)

--- a/usvm-jvm-instrumentation/src/main/kotlin/org/usvm/instrumentation/executor/InstrumentationProcessRunner.kt
+++ b/usvm-jvm-instrumentation/src/main/kotlin/org/usvm/instrumentation/executor/InstrumentationProcessRunner.kt
@@ -32,7 +32,7 @@ class InstrumentationProcessRunner(
         testingProjectClasspath: List<String>,
         jcClasspath: JcClasspath,
         instrumentationClassFactory: KClass<JcInstrumenterFactory<out JcInstrumenter>>
-    ) : this(testingProjectClasspath.joinToString(":"), jcClasspath, instrumentationClassFactory)
+    ) : this(testingProjectClasspath.joinToString(File.pathSeparator), jcClasspath, instrumentationClassFactory)
 
     fun isAlive() = this::lifetime.isInitialized && lifetime.isAlive
 

--- a/usvm-jvm-instrumentation/src/main/kotlin/org/usvm/instrumentation/executor/UTestConcreteExecutor.kt
+++ b/usvm-jvm-instrumentation/src/main/kotlin/org/usvm/instrumentation/executor/UTestConcreteExecutor.kt
@@ -10,6 +10,7 @@ import org.usvm.instrumentation.testcase.api.UTestExecutionResult
 import org.usvm.instrumentation.testcase.descriptor.UTestUnexpectedExecutionBuilder
 import org.usvm.instrumentation.util.InstrumentationModuleConstants
 import org.usvm.instrumentation.util.UTestExecutorInitException
+import java.io.File
 import kotlin.reflect.KClass
 import kotlin.time.Duration
 
@@ -25,7 +26,7 @@ class UTestConcreteExecutor(
         testingProjectClasspath: List<String>,
         jcClasspath: JcClasspath,
         timeout: Duration
-    ) : this(instrumentationClassFactory, testingProjectClasspath.joinToString(":"), jcClasspath, timeout)
+    ) : this(instrumentationClassFactory, testingProjectClasspath.joinToString(File.pathSeparator), jcClasspath, timeout)
 
     private val lifetime = LifetimeDefinition()
 

